src/ServicePulse.Host/app/js/views/failed_messages/view.html

```diff
@@ -50,88 +47,54 @@
                             </li>
                         </ul>
                     </div>
+
                 </div>
             </div>
         </div>
 
         <div ng-show="vm.failedMessages.length > 0" infinite-scroll="vm.loadMoreResults(vm.selectedExceptionGroup)" infinite-scroll-distance="0" infinite-scroll-disabled="vm.disableLoadingData">
+            <div class="col-sm-12 no-mobile-side-padding">
+                <div class="row box repeat-item failed-message" ng-repeat="message in vm.failedMessages">
+                    
+                    <div class="check col-xs-1" ng-click="vm.toggleRowSelect(message)" ng-mouseenter="message.hover = true" ng-mouseleave="message.hover = false" style="cursor: default;">
+                        <div class="checkbox" ng-class="{'check-hover': message.hover}">
+                            <input isolate-click type="checkbox" id="onoffswitch{{$id}}" name="onoffswitch{{$id}}" ng-disabled="message.retried || message.archived" ng-checked="message.selected" ng-click="vm.toggleRowSelect(message)">
```

Comment:
Something is visually wrong here:

![splunk-in-pulse](https://cloud.githubusercontent.com/assets/1060960/22964518/c65f2d4c-f361-11e6-870c-b0b6d4e61ae5.png)